### PR TITLE
Various test fixes

### PR DESCRIPTION
**Pull request by jorgebay**
Address recurrent CI failures.

### CODE REVIEW

File: test/integration/short/error-tests.js [hunk 1]
```diff
@@ -12,20 +12,21 @@ var vdescribe = helper.vdescribe;
...
       queries: [
         'CREATE TABLE read_fail_tbl(pk int, cc int, v int, primary key (pk, cc))',
+        helper.createKeyspaceCql(failWritesKs, 2, true),
+        helper.createTableCql(failWritesTable),
         helper.createKeyspaceCql('ks_func'),
         'CREATE TABLE ks_func.tbl1 (id int PRIMARY KEY, v1 int, v2 int)',
         'INSERT INTO ks_func.tbl1 (id, v1, v2) VALUES (1, 1, 0)',
-        "CREATE FUNCTION ks_func.div(a int, b int) RETURNS NULL ON NULL INPUT" +
-        " RETURNS int LANGUAGE java AS 'return a / b;'"
+        'CREATE FUNCTION ks_func.div(a int, b int) RETURNS NULL ON NULL INPUT RETURNS int LANGUAGE java AS' +
```
Comment (landim) [inline test/integration/short/error-tests.js]: single quote through all the code. 👍

File: test/integration/short/metadata-tests.js [hunk 2]
```diff
@@ -250,10 +250,10 @@ describe('Metadata', function () {
         client.connect.bind(client),
         function executeQuery(next) {
           client.execute(helper.queries.basic, [], { traceQuery: true}, function (err, result) {
-            traceId = result.info.traceId;
             if (err) {
               return next(err);
             }
+            traceId = result.info.traceId;
```
Comment (landim) [inline test/integration/short/metadata-tests.js]: nice catch!

File: test/unit/stream-id-stack-tests.js [hunk 3]
```diff
@@ -62,8 +63,8 @@ describe('StreamIdStack', function () {
     stack.clear();
   });
   it('should release unused groups', function (done) {
-    var releaseDelay = 50;
-    var stack = newInstance(3, releaseDelay - 10);
+    var releaseDelay = 100;
```
Comment (landim) [inline test/unit/stream-id-stack-tests.js]: I question myself if is it better to use a fixed value or a relative (percentage)

Comment (jorgebay) [inline test/unit/stream-id-stack-tests.js]: The `osPrecision` can be made into a helper property and reused, as any value greater than 15.6 (1000 / 64) will work on modern Windows, Linux and macOS.
Any interval greater than twice the precision for the delay would work, ie: `delay ± precision`

**Comment by jorgebay** [comment]
hmm... it looks like there are still test failures related to auth. I don't understand why its happening as `-Dcassandra.superuser_setup_delay_ms=0` is set, maybe its not being correctly passed??

I see on the C# driver that a similar setup, is not failing: https://github.com/datastax/csharp-driver/blob/master/src/Cassandra.IntegrationTests/Core/SessionAuthenticationTests.cs#L39-L60

The only difference is the 10s vs 5s additional wait time.

**Pull request merged**